src/components/Email/View.jsx

```diff
@@ -25,15 +25,23 @@ const EmailView = ({
           required={true}
           value={email}
           error={!error ? ["Please enter valid email"] : []}
-          placeholder="Enter your email"
+          placeholder={"Enter your email"}
           onChange={(id, value) => {
             setEmail(value);
             const re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
             const error = re.test(String(email).toLowerCase());
             setError(error);
-            if (error) {
-              setFormbuilderInputValue(path, blockid, value);
-            }
+            setFormbuilderInputValue(path, blockid, {
+              label: "Email",
+              validation: {
+                required: true,
+              },
+              placeholder: "Enter your email",
+              customErrormessage: "Please enter a valid email",
+              valid: error,
```

Comment:
if this is an error message it should say "error" as key. If it is a boolean value it should use "isValid" as key.

Reply:
It is not the error, it is the customErrorMessage which Editor provide during form building